spec/_layouts/default.html

```diff
@@ -1,13 +1,26 @@
 ---
 baseurl: "file:///Users/ehaas/Documents/FHIR/smart-app-launch-brver/output/"
```

Comment:
These paths seems to reference file locations on your own personal computer @Healthedata1  -- I wouldn't expect to see this in checked-in code.